.github/workflows/test.yml

```diff
@@ -79,6 +69,16 @@ jobs:
           - 9200:9200
     
     steps:
+      # See: https://remarkablemark.org/blog/2022/05/12/github-actions-postgresql-increase-max-connections-and-shared-buffers/
+      # See: https://stackoverflow.com/questions/70673766/how-to-increase-max-connection-in-github-action-postgres
```

Comment:
https://stackoverflow.com/a/78546472/8083927 seems much simpler. Does it work?

Reply:
No,it does not work. I've tries it.